ietfdata/datatracker.py

```diff
@@ -1070,6 +1104,63 @@ def meeting_types(self) -> Iterator[MeetingType]:
         """
         return self._retrieve_multi("/api/v1/name/meetingtypename/", MeetingType)
 
+# =================================================================================================================================
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?source=...      - documents that source draft relates to (references, replaces, etc)
+#   https://datatracker.ietf.org/api/v1/doc/relateddocument/?target=...      - documents that relate to target draft
+    
+    def related_documents_source(self,
```

Comment:
Can you combine these into a single `related_documents(self, source, dest, relation_type)` function?

Reply:
I have now combined both into a single function. Although, it may be too convoluted.